get-started/mcp/sdk-integrations/openai-agents.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Integrate OpenAI Agents SDK with kluster Verify
```

Comment:
This title is a bit long, but I'm going to let it go and see if we get flagged for it in our semrush site audit 🤷‍♀️